include/pmacc/pluginSystem/containsStep.hpp

```diff
@@ -36,7 +36,7 @@ namespace pmacc
          * @param timeStep simulation time step to check
          * @return true if step is included in the interval list else false
          */
-        HINLINE bool containsStep(std::vector<pluginSystem::Slice> const& seqTimeSlices, uint32_t const timeStep)
+        inline bool containsStep(std::vector<pluginSystem::Slice> const& seqTimeSlices, uint32_t const timeStep)
```

Comment:
offline answered question: `HINLINE` -> `inline` avoids including `pmacc/types.pp` which pulls full alpaka.

Reply:
And it does not matter here since it is CPU code anyways